src/mod_roster_mnesia.erl

```diff
@@ -108,6 +109,13 @@ remove_user_t(_HostType, LUser, LServer) ->
                   mnesia:index_read(roster, US, #roster.us)),
     ok.
 
+%% There is no optimized way to remove a domain.
+%% We expect, that domain removal process would call remove_user instead
+%% for each user.
+-spec remove_domain_t(mongooseim:host_type(), jid:lserver()) -> ok.
+remove_domain_t(_HostType, _LServer) ->
```

Comment:
Maybe better argument name will be `_Domain`. Then you can update `spec` as:
```erlang
-spec remove_domain_t(HostType :: mongooseim:host_type(), Domain :: jid:lserver()) -> ok.
```

Reply:
Argument name can be changed to `Domain`, but I don't think there is a need to repeat variable names in such a simple spec.